src/dialog.html

```diff
@@ -44,6 +44,11 @@
 			transform: rotateX(-90deg);
 		}
 
+		*[data-clickable="false"]{
+		    opacity: 0.3;
+		    cursor: not-allowed !important;
```

Comment:
The Origami CSS overrides this property. Moving the <link> to before the <style> removes the need for !important.